src/utils/contribsUtil/fetchRepoContribsUtil.js

```diff
@@ -10,6 +10,9 @@ function fetchRepoContributorsUtil(org, repo) {
 	  }else {
 		resolve(fetchRepoContribs.fetchRepoContribs(org, repo));
 	  }
+	}).catch((err) => {
+		console.log("err travelling through fetchRepoContribsUtil");
```

Comment:
oh this is great. Would there be another means of displaying errors to the user at some point? Thanks!

Reply:
@jywarren I've followed this comment: https://github.com/publiclab/community-toolbox/issues/284#issuecomment-554607740

Reply:
OK! I'll request a review from @Rishabh570 !